### PR TITLE
Sprint 20 merge

### PR DESCRIPTION
**Pull request by venkykandagaddala**

### CODE REVIEW

File: dist/.well-known/assetlinks.json [hunk 1]
```diff
@@ -7,18 +7,7 @@
             "namespace": "android_app",
             "package_name": "com.igot.karmayogibharat",
             "sha256_cert_fingerprints": [
-                "B0:A8:61:27:26:17:DD:68:30:07:C7:14:F7:C0:00:80:67:63:B8:50:88:E4:34:4A:46:36:C6:54:2E:31:CC:D4"
-            ]
-        }
-    },
-    {
-        "relation": [
-            "delegate_permission/common.handle_all_urls"
```
Comment (vishnubansaltarento) [inline dist/.well-known/assetlinks.json:16]: please check this file should not be part of PR

File: project/ws/app/src/lib/routes/events/routes/event-detail/event-detail.component.ts [hunk 2]
```diff
@@ -76,10 +76,9 @@ export class EventDetailComponent implements OnInit {
   }
 
   get isenrollFlow() {
-    if (this.eventData) {
+    if (this.eventData && this.enrollFlowItems.length) {
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/events/routes/event-detail/event-detail.component.ts:79]: please do check for this.enrollFlowItems first then length

Comment (venkykandagaddala) [reply project/ws/app/src/lib/routes/events/routes/event-detail/event-detail.component.ts:79]: enrollFlowItems is defined as array

File: dist/.well-known/assetlinks.json (hunk 1 above)
Comment (venkykandagaddala) [reply dist/.well-known/assetlinks.json:16]: @vishnubansaltarento Pawan from mobile team is added this. He confirmed that we need that change

**Pull request merged**